components/search/DateFilter.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <b-form inline>
```

Comment:
This component will be embedded in a form already, rendered by an ancestral component, so does not need its own form element.